screen.c

```diff
@@ -60,16 +71,30 @@ screen_reinit(struct screen *s)
 	grid_clear_lines(s->grid, s->grid->hsize, s->grid->sy, 8);
 
 	screen_clear_selection(s);
+
+	while ((title_entry = TAILQ_FIRST(s->titles)) != NULL) {
+		TAILQ_REMOVE(s->titles, title_entry, entry);
```

Comment:
Also, move this line *after* you'e free()d the other entries below, please.

Reply:
I'm sorry, I don't understand. Move which line below where? (I can't move the `TAILQ_REMOVE` line below `free(title_entry)`.)